lib/redux/reducers/users.js

```diff
@@ -87,6 +88,18 @@ export function updateProfileErrorDismissed(state = null, action) {
   }
 }
 
+export function createCustodialAccountErrorMessage(state = null, action) {
+  switch (action.type) {
+    case actionTypes.CREATE_CUSTODIAL_ACCOUNT_FAILURE:
+      const err = action.payload;
+      return err.message;
+    case actionTypes.CREATE_CUSTODIAL_ACCOUNT_REQUEST:
```

Comment:
This doesn't seem sufficient for clearing these errors. If I simulate an error on custodial account creation, I get the big red error rendered. I navigate away & back again, and it's still there. Not sure I would ever try again with that error showing. Was making the error dismissable through direct user action (rendering an 'x' in the corner) or dismissing it "under the hood" when user navigates away to another "page" discussed?